airflow/providers/google/cloud/triggers/dataproc.py

```diff
@@ -140,24 +153,75 @@ def serialize(self) -> tuple[str, dict[str, Any]]:
                 "gcp_conn_id": self.gcp_conn_id,
                 "impersonation_chain": self.impersonation_chain,
                 "polling_interval_seconds": self.polling_interval_seconds,
+                "delete_on_error": self.delete_on_error,
             },
         )
 
     async def run(self) -> AsyncIterator[TriggerEvent]:
-        while True:
-            cluster = await self.get_async_hook().get_cluster(
-                project_id=self.project_id, region=self.region, cluster_name=self.cluster_name
+        """Run the trigger."""
```

Comment:
```suggestion
```

Reply:
I don't think that docstring adds much value unless we need it for some pre-commit. WDYT?

Reply:
Removed it